packages/firestore/src/api/database.ts

```diff
@@ -216,14 +216,26 @@ export class Firestore
   }
 
   settings(settingsLiteral: PublicSettings): void {
+    const currentSettings = this._delegate._getSettings();
+    if (
+      !settingsLiteral.merge &&
+      currentSettings.host !== settingsLiteral.host
+    ) {
+      logWarn(
+        'The original host is being overridden. If you did not intend ' +
```

Comment:
Maybe change to "You are overriding the original host." to make it easier for us foreigners.

Reply:
Done.